.github/workflows/build.yml

```
@@ -64,12 +68,14 @@ jobs:
       with:
         name: ${{ matrix.container_tag }}
         path: config.log
+    # The configure script relies on the old syntax to check whether functions
+    # exist, so we can't specify `-Wstrict-prototypes` globally.
```

Maybe better place for this comment is in the configure step:
```suggestion
    # Configure scripts rely on the K&R syntax to check whether functions
    # exist, so `-Wstrict-prototypes` can be specified only for make.
```